lib/ingram_micro/transmission.rb

```diff
@@ -28,6 +28,20 @@ def schema_valid?
     valid
   end
 
+  # This will find the appropriate xsd file and return it for use in xml
+  # validation. For sales-order-submission, there are two different schemas,
+  # and which one to load depends on whether you need the one configured for
+  # domestic-only use or the one for mixed domestic and internatinal use.
+  def load_schema
```

Comment:
you're probably right, but i didn't write this method and i'm hesistant to change it without knowing why @vietjew did it like this, perhaps there was a reason for it